docs/DeveloperGuide.adoc

```diff
@@ -294,18 +294,31 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
+
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |manage and customise different resumes |use different resumes for different companies
+
+|`* * *` |user |create a resume file |print it
+
+|`* * *` |user |add to and remove 'items' from a certain resume |
```

Comment:
Can we specify 'items' a bit more? Such as each item is a resume field of type `pd, proj, edu, ski, int, res, achv`

Reply:
Added as comment at the bottom